pkgs/development/libraries/leveldb/default.nix

```
@@ -1,4 +1,4 @@
-{ stdenv, fetchFromGitHub, fixDarwinDylibNames }:
+{ snappy, stdenv, fetchFromGitHub, fixDarwinDylibNames }:
```

```suggestion
{ stdenv, fetchFromGitHub, fixDarwinDylibNames, snappy }:
```
Usually we leave the stdenv/helpers first.